<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtn_principal_cargar</class>
 <widget class="QMainWindow" name="vtn_principal_cargar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bienvenidos a Persona Cargar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tb_personas">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>621</width>
      <height>281</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cedula</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sexo</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_actualizar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Actualizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_refrescar">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Refrescar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_nuevo">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>320</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Nueva Persona</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_eliminar">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mnb_menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>694</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="stb_estado"/>
 </widget>
 <resources/>
 <connections/>
</ui>
